desktop/src/main/java/bisq/desktop/main/dao/wallet/send/BsqSendView.java

```diff
@@ -306,43 +379,80 @@ private void addSendBtcGroup() {
-
-        sendBtcButton.setOnAction((event) -> {
-            if (GUIUtil.isReadyForTxBroadcastOrShowPopup(p2PService, walletsSetup)) {
-                String receiversAddressString = receiversBtcAddressInputTextField.getText();
-                Coin receiverAmount = bsqFormatter.parseToBTC(btcAmountInputTextField.getText());
-                try {
-                    Transaction preparedSendTx = bsqWalletService.getPreparedSendBtcTx(receiversAddressString, receiverAmount);
-                    Transaction txWithBtcFee = btcWalletService.completePreparedSendBsqTx(preparedSendTx);
-                    Transaction signedTx = bsqWalletService.signTx(txWithBtcFee);
-                    Coin miningFee = signedTx.getFee();
-
-                    if (miningFee.getValue() >= receiverAmount.getValue())
-                        GUIUtil.showWantToBurnBTCPopup(miningFee, receiverAmount, btcFormatter);
-                    else {
-                        int txVsize = signedTx.getVsize();
-                        showPublishTxPopup(receiverAmount,
-                                txWithBtcFee,
-                                TxType.INVALID,
-                                miningFee,
-                                txVsize, receiversBtcAddressInputTextField.getText(),
-                                btcFormatter,
-                                btcFormatter,
-                                () -> {
-                                    receiversBtcAddressInputTextField.setText("");
-                                    btcAmountInputTextField.setText("");
-                                });
-
-                    }
-                } catch (BsqChangeBelowDustException e) {
-                    String msg = Res.get("popup.warning.btcChangeBelowDustException", btcFormatter.formatCoinWithCode(e.getOutputValue()));
-                    new Popup().warning(msg).show();
-                } catch (Throwable t) {
-                    handleError(t);
-                }
+        Tuple2<Button, Button> tuple = FormBuilder.add2ButtonsAfterGroup(root, ++gridRow,
+                Res.get("dao.wallet.send.sendBtc"), Res.get("dao.wallet.send.inputControl"));
+        sendBtcButton = tuple.first;
+        btcInputControlButton = tuple.second;
+
+        sendBtcButton.setOnAction((event) -> onSendBtc());
+        btcInputControlButton.setOnAction((event) -> onBtcInputControl());
+    }
+
+    private void onBtcInputControl() {
+        List<TransactionOutput> unspentTransactionOutputs = bsqWalletService.getSpendableNonBsqTransactionOutputs();
+        if (btcUtxoCandidates == null) {
+            btcUtxoCandidates = new HashSet<>(unspentTransactionOutputs);
+        } else {
+            // If we had some selection stored we need to update to already spent entries
+            btcUtxoCandidates = btcUtxoCandidates.stream().
+                    filter(e -> unspentTransactionOutputs.contains(e)).
+                    collect(Collectors.toSet());
```

Comment:
Same comment as for the BSQ, could be confusing if the selection changes without the users knowing about it

Reply:
If you make a send tx and spend a previous selected utxo we need to remove that. We always show that spendable utxos and those are marked the user has selected. A previous selected which got spent need to be removed.